api/spec/controllers/spree/api/v1/products_controller_spec.rb

```diff
@@ -399,7 +399,7 @@ module Spree
         end
       end
 
-      it 'can delete a product' do
+      xit 'can delete a product' do
```

Comment:
@KacperMekarski why are you xiting this tests?

Reply:
They were failing at the beginning, I see they are passing now, I will uxit them.

Reply:
@damianlegawiec done!
Everything seems to be ready.